features/support/worlds/forms_world.rb

```diff
@@ -0,0 +1,23 @@
+module FormsWorld
+
+  def fill_in_employer_registration_form
+    fill_in 'agency_organization_legal_name', with: registering_employer.legal_name
+    fill_in 'agency_organization_dba', with: registering_employer.dba
+    fill_in 'agency_organization_fein', with: registering_employer.fein
+    select 'Tax Exempt Organization', from: 'agency_organization_entity_kind'
+    #select "0111", from: "agency_organization_profile_attributes_sic_code"
```

Comment:
Were you having issues here being unable to choose the sic code? I'm wondering if its because the select contains optgroups.

Reply:
@ZASMan  At first yes, but it was because config.action_controller.allow_forgery_protection = true was initially set to false which prevent axios from making http request in the test env.  See https://github.com/health-connector/enroll/commit/b96b2b728f551038720a42f2e79cba450bd83a93